src/components/HistoryDialog/index.tsx

```diff
@@ -15,14 +15,14 @@ export const HistoryDialog = () => {
   const { isReady } = useAssets();
 
   const entries = useMemo(() => {
-    return isReady ? Object.entries(history) : undefined;
+    return isReady ? [...Object.entries(history)].reverse() : undefined;
```

Comment:
Oh you know what, I did it because `.reverse()` modifies the array, but now that you say it it doesn't matter because it's acting on `Object.entries(history)` rather than `history` itself. Good catch